docs/architecture/notes.md

```diff
@@ -177,12 +179,12 @@ _Info: To compute a note's hash, we do not need to know the note's `serial_num`.
 
 ## Note discovery
 Note discovery describes the process of Miden clients finding notes they want to consume. There are two ways to receive new relevant notes - getting notes via an off-chain channel or querying the Miden operator to request newly recorded relevant notes.
-The latter is done via note tags. Tags are part of the note's metadata and are represented by a `Felt`. The `SyncState` API of the [Miden node](../network/miden-node.md) requires the Miden client to provide a `note_tag` value which is used as a filter in the operator's response. Tags are useful for note discovery enabling an easy collection of all notes matching a certain tag.
+The latter is done via note tags. Tags are part of the note's metadata and are represented by a `Felt`. The `SyncState` API of the Miden node requires the Miden client to provide a `note_tag` value which is used as a filter in the operator's response. Tags are useful for note discovery enabling an easy collection of all notes matching a certain tag.
```

Comment:
there's a whole bunch of these ... previously, in an earlier go at this, I made them an unworking link so it would be easy to see but was asked to remove them in a previous PR ... so that's why I removed it ...

All these should be dealt with on an indepth review.